bigbluebutton-html5/imports/ui/components/presentation/service.js

```diff
@@ -23,15 +23,6 @@ const getCurrentSlide = (podId) => {
       presentationId: currentPresentation.id,
       current: true,
     },
-    {
-      fields: {
-        meetingId: 0,
-        thumbUri: 0,
-        swfUri: 0,
-        txtUri: 0,
```

Comment:
Should we not simply switch `txtUri: 0` to `1`? Looks like we could still exclude the other properties

Reply:
I tried, but received this error:
`You cannot currently mix including and excluding fields`